YOLONAS.md

```diff
@@ -32,6 +32,21 @@ import super_gradients
 
 yolo_nas = super_gradients.training.models.get("yolo_nas_l", pretrained_weights="coco").cuda()
 yolo_nas.predict("https://deci-pretrained-models.s3.amazonaws.com/sample_images/beatles-abbeyroad.jpg").show()
+```
+## Extract bounding boxes 
+```python
+
+output = yolo_nas.predict("https://deci-pretrained-models.s3.amazonaws.com/sample_images/beatles-abbeyroad.jpg")
+'''
+output._images_prediction_lst = [ImageDetectionPrediction(image=array([[[.,.,.],[.,.,.],[.,.,.]...],...],dtype=float32),
+                                prediction=DetectionPrediciton(bboxes_xyxy=array([],shape(0,4),dtype=float32),confidence=array([],dtype=float32),labels=array([],dtype=float32)),class_names=['name','of','all','classes'])]
+'''
```

Comment:
I believe that with the example on how to get each attribute that you added below, this is redundant so I think it's better to only keep the example, and remove this string from the tutorial